docs/guides/03-custom-deployments.mdx

```diff
@@ -272,3 +272,13 @@ forge script script/DeployFlow.s.sol \
 
 If you are using a mnemonic or a hardware device for your deployer address, refer to `forge-script` page from
 [foundry book](https://book.getfoundry.sh/reference/forge/forge-script#forge-script) for different wallet options.
+
+### Steps for listing the deployments
+
+Once the contracts are deployed, the final step is to list the deployment addresses and artifacts in the Lockup docs:
```

Comment:
Why 'Lockup docs' and not just 'docs'?

Reply:
https://github.com/sablier-labs/docs/pull/214/commits/65c2f4f707eba738ac4902c089e4106cdac6cf50